### ISSUE TITLE
[Adaptive Cards> RestaurantOrder.json]: On receiving the keyboard focus on 'Which side would you like' edit field suggestion list is getting automatically opened.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8633]
### Target Platforms

UWP

### SDK Version

1.2.2306.30005

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 23H2 (Build 25905.1000)
Application: AdaptiveCardVisualizer
Application Version: 1.2.2306.30005
Screen reader: Narrator

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Turn on the narrator through Ctrl+ Windows+ Enter key.
3. Navigate to the 'RestaurantOrder.json' through tab key and activate it.
4. Navigate to which side would you like edit field through tab key.
5. Observe the issue.

**Actual Result:**
On receiving the keyboard focus on 'Which side would you like' edit field suggestion list is getting automatically opened.

**Expected Result:***
On receiving the keyboard focus on 'Which side would you like' edit field suggestion list is should not get automatically opened.
Suggestion list should get opened when user type any character in the edit field or its role should be defined as combobox if only these suggestions needs to be selected.

**Notes:**
Same issue repro with 'Which entree would you like?' and 'Which drink would you like?' edit field.
When user navigates on the suggestion list as focus lands on the first suggestion list it gets automatically selected.

**User Impact**:
Keyboard dependent user will get impacted as user will not be able to interact or navigate the content properly.

**Reference link:**
https://www.w3.org/WAI/WCAG21/Understanding/on-focus

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards/assets/104483005/2f046d1a-8191-4521-b040-4551a687c029

### Card JSON

```JSON
NA
```
